x/feemarket/keeper/feemarket.go

```diff
@@ -22,7 +23,7 @@ func (k *Keeper) UpdateFeeMarket(ctx sdk.Context) error {
 
 	state, err := k.GetState(ctx)
 	if err != nil {
-		return err
+		panic(err)
```

Comment:
If we panic here the chain will halt right? Shouldn't we return the error without applying the changes and let the parent decide if it should panic?

Reply:
I see your point, it makes sense